pages/learn/deploy/offline-updates.md

```diff
@@ -26,7 +26,7 @@ When you insert the SD card or USB drive into your device and boot it, the devic
 
 ## Configuration and Data Recovery
 
-Some consideration is required if an application requiring persistent data storage is being used to recover user data while using the offline update process. The application can't use the pre-provisioned [named volumes][named-volumes], since these will be wiped during the offline update process.
+Some consideration is required if a fleet's requiring persistent data storage is being used to recover user data while using the offline update process. The fleet's can't use the pre-provisioned [named volumes][named-volumes], since these will be wiped during the offline update process.
```

Comment:
```suggestion
Some consideration is required if an application requiring persistent data storage is being used to recover user data while using the offline update process. The application can't use the pre-provisioned [named volumes][named-volumes], since these will be wiped during the offline update process.
```